### ISSUE TITLE
[REQUEST] Block\NOTBlock IP by port used

### ISSUE THREAD

**Issue by GoogleCodeExporter** [soul4soul/peerblock#383]
```
HI,
i have some problem using TOR (I am a relay) & all my incomming connection on 
port 443 & 9030 al filtered by the bluetack lists.
Can you add a feature for UNBlock these ports & block other port 
(3010.3000,3020 used by bad edonkey cient)? 
I can thest the new features

thanks
**Riso64Bit**

```

Original issue reported on code.google.com by `Riso64Bit` on 13 Feb 2011 at 10:31
- Merged into: #12

**Comment by GoogleCodeExporter** [comment]
Original comment by `nightstalkerz` on 30 May 2011 at 9:37
- Changed state: **Duplicate**
